etstool.py

```diff
@@ -385,6 +386,38 @@ def test_all(edm):
         sys.exit(1)
 
 
+@cli.command(name="generate-parser")
+@edm_option
+@runtime_option
+@click.option(
+    "--environment", default=None, help="Name of EDM environment to use."
+)
+def generate_parser(edm, runtime, environment):
+    """ Regenerate the mini-language parser for the observe framework.
+    """
+    parameters = get_parameters(edm, runtime, environment)
+
+    root_dir = os.path.dirname(__file__)
+    observers_dir = os.path.join(root_dir, "traits", "observers")
+
+    # parser file to be generated.
+    out_path = os.path.join(observers_dir, "_generated_parser.py")
+
+    # grammar file.
+    parameters["grammar_path"] = os.path.join(
+        observers_dir, "_dsl_grammar.lark")
+
+    command = (
+        "{edm} run -e {environment} -- "
+        "python -m lark.tools.standalone "
+        "{grammar_path}"
+    )
+
+    with open(out_path, "w") as out_file:
```

Comment:
Is there a command-line option to get `lark` to output to a file?  that would be preferable to capturing stdout.

Reply:
I checked and unfortunately no:
https://github.com/lark-parser/lark/blob/master/lark/tools/standalone.py#L87

Well spotted about the source distribution.  I will add the file to MANIFEST.in